### DISCUSSION TITLE
How to disassemble new 2021 square model (S100103, Lay-Z Helsinki)

### DISCUSSION THREAD

**Discussion by hubaksis** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#161] [Q&A] [answered]
Hi there,
Do you have any pictures how to carefully disassemble 201 square unit?
I have S100103 unit for Lay-Z Helsinki. Wanted to check the number of wires, and don't want to break the display. It doesn't fall apart after taking 6 screws behind the display. 

Do I need to disassemble the whole unit to get to the display?

Appreciate any help :)

Cheers

**Comment by NitramH** [comment]
Hi,

I have the Miami and had the same problem. You need to open the big cube first,
Then you see the number of cables. If you want to open the display too,
you need to access some hidden screws from the inner side of the big cube.

**Reply by hubaksis** [reply]
Oh, so there is no need to disassemble the display.
I'll try to make a couple of photos and post them.
Thanks for the tip!

**Comment by hubaksis** [comment, marked as answer]
To disassemble this unit you need to unscrew 6 screws as on the pictures below.
![IMG_20211231_150908](https://user-images.githubusercontent.com/13341477/148151442-86728fd2-0616-4674-b1bd-bd75785351da.jpg)
![IMG_20211231_150915](https://user-images.githubusercontent.com/13341477/148151456-d6263234-7459-47d1-ad8c-151665b8ca56.jpg)
![IMG_20211231_150920](https://user-images.githubusercontent.com/13341477/148151462-03b86412-a9ac-4c72-9fa9-d9e3e90b7576.jpg)
![IMG_20211231_150925](https://user-images.githubusercontent.com/13341477/148151466-b40f3d4f-8514-42ab-a9b2-ad91efe6e0d7.jpg)

**Reply by kgreene1983** [reply]
so are the connectors in there? Room to hide the pcb?

**Reply by hubaksis** [reply]
Yes, the connector is in there.
Plenty of room to hide PCB. I think water should not be there, but I put PCB in the box just in case.

![image](https://user-images.githubusercontent.com/13341477/152623167-b338481e-d548-47f5-b02f-8ee8466fa82a.png)

**Reply by hjkoster400d** [reply]
Thanks, this is very helpful. @hubaksis Did you need to install the additional 560-680 ohm resistor between the LLC and display? PCB is on it's way, so i will be glad to be prepared ;)

**Reply by hubaksis** [reply]
The documents recommended resistors between the WiFi device and the display, so I installed them. I don't remember checking if it works without them

![image](https://user-images.githubusercontent.com/13341477/175240294-c499391a-431e-45df-a347-7355cf3dead3.png)
